src/lib/esnext.string.d.ts

```diff
@@ -0,0 +1,11 @@
+interface String {
+    /**
+     * Returns true if the string does not contain any lone UTF-16 surrogates.
+     */
+    isWellFormed(): boolean;
+
+    /**
+     * Returns a string where all lone UTF-16 surrogates of this string have been replaced by the Unicode replacement character (U+FFFD).

```

Comment:
Same here: 

```suggestion
     * Returns a string where all lone or out-of-order UTF-16 surrogates have been replaced by the Unicode replacement character (U+FFFD).
```

I'm not sure whether UTF-16 needs to be mentioned or not.